ix-dev/community/esphome/templates/docker-compose.yaml

```diff
@@ -10,7 +10,6 @@
 {% do c1.healthcheck.set_test("curl", {"port": values.network.web_port.port_number, "path": "/version"}) %}
 {% do c1.set_command(["dashboard", values.consts.config_path, "--port", values.network.web_port.port_number]) %}
 
-{% do c1.environment.add_env("ESPHOME_DASHBOARD_USE_PING", true) %}
```

Comment:
Lets make it configurable? defaulting to true?

Reply:
I think if we can make it work without it, the default should be false. People can override it with an environment variable.